e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -492,55 +492,57 @@ describeEE("scenarios > embedding > full app", () => {
           },
         });
       });
-      cy.get("@postMessage")
-        .should("have.been.calledWith", {
+
+      // TODO: Find a way to assert that this is the last call.
+      cy.get("@postMessage").should(
+        "have.been.calledWith",
+        {
           metabase: {
             type: "frame",
             frame: {
               mode: "fit",
               height: Cypress.sinon.match(value => value > 1000),
             },
           },
-        })
-        .and("not.have.been.calledWith", {
+        },
+        "*",
+      );
+
+      cy.get("@postMessage").invoke("resetHistory");
+      cy.findByRole("tab", { name: TAB_2.name }).click();
+      cy.get("@postMessage").should(
+        "have.been.calledWith",
+        {
           metabase: {
             type: "frame",
             frame: {
               mode: "fit",
               height: Cypress.sinon.match(value => value < 400),
             },
           },
-        });
-
-      cy.findByRole("tab", { name: TAB_2.name }).click();
-      cy.get("@postMessage").should("have.been.calledWith", {
-        metabase: {
-          type: "frame",
-          frame: {
-            mode: "fit",
-            height: Cypress.sinon.match(value => value < 400),
-          },
         },
-      });
+        "*",
+      );
 
+      cy.get("@postMessage").invoke("resetHistory");
       cy.findByTestId("app-bar").findByText("Our analytics").click();
 
       cy.findByRole("heading", { name: "Metabase analytics" }).should(
         "be.visible",
       );
-      cy.get("@postMessage").then(postMessage => {
-        expect(
-          postMessage.lastCall.calledWith({
-            metabase: {
-              type: "frame",
-              frame: {
-                mode: "fit",
-                height: 800,
-              },
+      cy.get("@postMessage").should(
```

Comment:
Change this because it seems to always fail when stress testing. I don't know why. All calls actually have a second argument `"*"`, but it never failed when stress testing. Only the last assertion failed.

- [PR stress tests](https://github.com/metabase/metabase/actions/runs/9368575231)
- [Master stress tests](https://github.com/metabase/metabase/actions/runs/9368577269)